p4include/v1model.p4

```diff
@@ -56,23 +56,51 @@ struct standard_metadata_t {
     //
     // flattening fields that exist in bmv2-ss
     // queueing metadata
-    @alias("queueing_metadata.enq_timestamp") bit<32> enq_timestamp;
-    @alias("queueing_metadata.enq_qdepth")    bit<19> enq_qdepth;
-    @alias("queueing_metadata.deq_timedelta") bit<32> deq_timedelta;
-    @alias("queueing_metadata.deq_qdepth")    bit<19> deq_qdepth;
+    @alias("queueing_metadata.enq_timestamp")
+    bit<32> enq_timestamp;
+    @alias("queueing_metadata.enq_qdepth")
+    bit<19> enq_qdepth;
+    @alias("queueing_metadata.deq_timedelta")
+    bit<32> deq_timedelta;
+    /// queue depth at the packet dequeue time.
+    @alias("queueing_metadata.deq_qdepth")
+    bit<19> deq_qdepth;
+
     // intrinsic metadata
-    @alias("intrinsic_metadata.ingress_global_timestamp") bit<48> ingress_global_timestamp;
-    @alias("intrinsic_metadata.egress_global_timestamp") bit<48> egress_global_timestamp;
-    @alias("intrinsic_metadata.lf_field_list") bit<32> lf_field_list;
-    @alias("intrinsic_metadata.mcast_grp")     bit<16> mcast_grp;
-    @alias("intrinsic_metadata.resubmit_flag") bit<32> resubmit_flag;
-    @alias("intrinsic_metadata.egress_rid")    bit<16> egress_rid;
+    @alias("intrinsic_metadata.ingress_global_timestamp")
+    bit<48> ingress_global_timestamp;
+    @alias("intrinsic_metadata.egress_global_timestamp")
+    bit<48> egress_global_timestamp;
+    /// Learn filter field list
+    @alias("intrinsic_metadata.lf_field_list")
+    bit<32> lf_field_list;
+    /// multicast group id (key for the mcast replication table)
+    @alias("intrinsic_metadata.mcast_grp")
+    bit<16> mcast_grp;
+    /// flag distinguishing original packets from resubmitted packets.
+    @alias("intrinsic_metadata.resubmit_flag")
+    bit<32> resubmit_flag;
+    /// Replication ID for multicast
+    @alias("intrinsic_metadata.egress_rid")
+    bit<16> egress_rid;
+    @alias("intrinsic_metadata.recirculate_flag")
+    bit<32> recirculate_flag;
     /// Indicates that a verify_checksum() method has failed.
-    // 1 if a checksum error was found, otherwise 0.
+    /// 1 if a checksum error was found, otherwise 0.
     bit<1>  checksum_error;
-    @alias("intrinsic_metadata.recirculate_flag") bit<32> recirculate_flag;
     /// Error produced by parsing
     error parser_error;
+    /// set packet priority
+    @alias("intrinsic_metadata.priority")
+    bit<3> priority;
+
+    // Queueing metadata
+    /// flag indicating whether a packet can be deflected by TM on congestion drop
+    @alias("intrinsic_metadata.deflect_on_drop")
+    bit<1> deflect_on_drop;
+    /// time snapshot taken when the packet is enqueued (in nsec).
+    @alias("intrinsic_metadata.enq_tstamp")
+    bit<32> enq_tstamp;
```

Comment:
This was said during LDWG meeting, but my main comment is that I hope that the rest of these changes still pass all tests if you remove deflect_on_drop and enq_tstamp from this struct definition.  I have tried to remove all uses of those fields from all p4c test programs, so I have some hope that this is true.  I am happy to analyze whatever fails, if anything fails, and get back to you quickly if I see a way to further edit the test programs to eliminate the use of these fields.

Reply:
I have run p4c tests locally on my dev machine with this PR's changes, plus removing deflect_on_rop and enq_tstamp.  The only failing tests are due to line number changes in 2 error messages involving v1model.p4 line numbers (expected, and easily fixed in this PR), and about a dozen p4test programs that have different mid-end P4_16 source code because of these 2 removed fields, again expected and easily fixed.

Reply:
I will go ahead and approve this, and if my suggested changes aren't made before merging this PR, I can create another one afterwards that makes them.